setup.py

```diff
@@ -11,4 +12,4 @@
       url='http://www.arcoslab.org/',
       package_dir={'pyrovito': ''},
       packages=['pyrovito'],
-      scripts=['pyrovito'])
+      scripts=['pyrovito', 'vis_sink.py'])
```

Comment:
Since this is new code I'd rather you used a ` console_scripts` entry points ([link to docs](https://setuptools.readthedocs.io/en/latest/setuptools.html#automatic-script-creation)) instead of the old `scripts` way. If you follow the file struture suggestion it could be something like this:

```python
    entry_points={
        'console_scripts': [
            'vis_sink'='pyrovito.scripts.vis_sink:main'
           ' pyrovito'='pyrovito.scripts.pyrovito:main' 
        ]
},
```
And move the top level instructions (the ones with no indent) to a `main` function

Reply:
I don't understand what is the point with this. What to we gain? just removing the .py at the end?

Reply:
I would like to clarify you could remove the `.py` form the script name yourself if you wanted and it is a minor detail, it is nice since *NIX shell executables by convension don't use it, but it is a minor detail.

1. `console_scripts` require minimal changes to be supported, the directory structure that I suggest is just that, a suggestion. All you need to do is put the top level module instructions on a function. 
2. `console_scripts` entry point manages a few things that `scripts` doesn't regarding operating system interoperability. You could argue that this is not important for us since we only use Debian, and you'd be right, however this is also a free software project and it might be useful for someone else and it takes little effort to support doing this the right way. Similarly, I think doing things that brings extra _unsued_ advantages that bring no disadvantages with them is useful since we might use them or need them in the future, specially for low effort things like this one.
3. `console_scripts`  IS right way™️ to do this. `console_scripts` is the recommended way to provide executable scripts and in line with best practices regarding packaging in the python community, which we, ARCOS-Lab, internally agreed to follow, this alone should be enough argument.

4.  `script` is actually a legacy feature. It won't probably be removed in the future so it's save to keep using, but given the chance, and in general, prefer coding using with non-legacy/recommended features unless there is a justification not to. Given the low effort for the change I don't think that using the legacy `scripts` feature is justified.

Hope this clears my positions regarding this.